src/database/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+pub mod database;
+pub mod enums;
+pub mod math;
+pub mod output;
+mod sql_structs;

```

Comment:
I don't think it matters too much, but why isn't this `pub mod`?

Reply:
We don't want to expose any of the sql definitions outside of the crate, it results in unwanted auto imports